### DISCUSSION TITLE
Quale allarme ?

### DISCUSSION THREAD

**Discussion by guestisp** [indomus/forum#2447] [GENERALE - Discussioni generiche]
Ciao ragazzi.
Voi quale sistema di allarme acquistereste, oggigiorno ? Ne ho uno piuttosto vecchio in casa che mi sta dando qualche grattacapo, vorrei cambiarlo con qualcosa di moderno, ma non ci capisco niente.

Ho visto ialarm e casasicura, prodotti sostanzialmente identici, se non fosse che ialarm PARE avere la multifrequenza simultanea
Non mi convince affatto il fatto che siano gestibili solo via cloud, se dovessero fallire, il mio allarme sarebbe inutilizzabile.

Avete qualche idea voi ?

**Comment by gebtig01** [comment]
Ciao, 
anche io sono curioso dei feedback della community. Ho installato iAlarmXR in diverse case di amici e funziona davvero molto bene. 
Potrebbero fare un restyling dell’app (in stile Ajax) ma comunque nulla da dire. Qualora non dovesse andare il cloud c’è sempre la pulsantiera, la SIM gsm e i vari tag di sblocco/inserimento totale.
Anche soluzioni più “accattivanti” come ajax.system sono gestite via cloud (credo) in modo analogo, quindi quello che poni è un discorso diffuso.
Sono interessato al parere altrui, quindi spero che il thread prosegua.

**Comment by indomus** [comment]
In attesa di feedback di altri, ecco un contributo per chi legge: https://indomus.it/focus/panoramica-i-sistemi-dallarme-la-domotica-e-home-assistant/

**Comment by guestisp** [comment]
contattati più volte telefonicamente ed anche a mezzo email, mi hanno confermato che il Cloud, nel loro caso, non è obbligatorio, la centrale si può gestire in autonomia tramite app anche mediante rete locale senza accesso ad internet. un componente per HA viene fornito post-vendita (vediamo se è vero, ho ricevuto il kit oggi, ma non ho HA al momento)

**Comment by guestisp** [comment]
piccolo update: ricevuto ora il kit. il manuale, sia quello incluso nella scatola, sia quello dettagliato che fanno scaricare online, sembra davvero ben fatto e curato graficamente. (abituato ai manuali delle cinesate, questo è un lusso).

Nel manuale effettivamente è scritto che si può usare la centrale in modalità locale, attivando il suo access point interno. Bisogna vedere se la si connette al wifi locale (invece di usare l'AP interno), è poi possibile usare l'app specificando l'ip che si prende la centrale (l'app chiede appunto l'ip di collegamento, locale in caso di AP, cloud alibaba in germania in caso di modalità online. Secondo me, proverò oggi pomeriggio se ho tempo, è possibile indicare l'ip diretto della centrale ottenuto via dhcp)

Tutto sommato, non mi sembra malvagio come prodotto. Chiaramente non lo si usa per proteggere una banca, ma secondo me come allarme da casa può fare il suo lavoro.

**Comment by gebtig01** [comment]
Ti confermo che il componente custom funziona bene fino a HA 2023.4.x.
La 2023.5 ha introdotto diverse modifiche e mi sembra che abbia perso
compatibilità, uscirà un aggiornamento immagino 🤞
Facci sapere appena antifurto365 ti manda lo zip.

Se riesco provo anche io oggi puntando direttamente all’IP locale della
centrale, ma non credo…

Il giorno ven 5 mag 2023 alle 09:49 GUEST.it ***@***.***> ha
scritto:

**Comment by guestisp** [comment]
no, una volta connessa ad internet, la centrale non ha alcuna porta attiva. la 18034 è presente solo in modalità AP che però si stacca da sola dopo mezz'ora circa

qualcuno ha idea di come funzioni il loro server ed in che modo dialoga con la centrale? perché potrebbe essere più semplice farsi in casa un gateway dove fare puntare l'app.
